src/Backend/App.vala

```diff
@@ -59,6 +68,8 @@ public class Slingshot.Backend.App : Object {
         keywords = info.get_keywords ();
         categories = info.get_categories ();
         generic_name = info.get_generic_name ();
+        prefers_non_default_gpu = !info.get_boolean ("PrefersNonDefaultGPU");
```

Comment:
As the boolean is inverted here, I would prefer if this field wasn't called `prefers_non_default_gpu`, because it's the opposite of that

Reply:
Fixed!